src/main/java/frc/robot/subsystems/Dj.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.ColorSensorV3;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj2.command.*;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+
+import static com.revrobotics.CANSparkMaxLowLevel.MotorType.kBrushless;
+import static frc.robot.Constants.IntakeConstants.*;
+
+public class Dj extends SubsystemBase {
+
+  private final CANSparkMax DJMotor = new CANSparkMax(k_DJ_MOTOR_ID, kBrushless);
+  private final DigitalInput button = new DigitalInput(BUTTON_CHANNEL);
+  private final DigitalInput beamBreaker = new DigitalInput(BEAMBREAK_CHANNEL);
+  private final ColorSensorV3 colorSensorV3 = new ColorSensorV3(I2C.Port.kMXP);
+
+  private final Trigger beamBreakDetected = new Trigger(beamBreaker::get);
+  //when beam brakers are connected its
```

Comment:
it's what?
also an unnecessary comment.
you only handle the sensor directly in the `Trigger` declaration.
make sure the `Trigger` has an intuitive name